documentation/analytics.md

```diff
@@ -5,13 +5,32 @@ about how to improve it. You can opt out of providing this data.
 
 ## Opt out of data collection
 
-You can opt out of data collection by modifying Bolt's analytics configuration
-file. This file is located in the user's home directory at `<HOME
```

Comment:
We also need to update https://github.com/puppetlabs/bolt/pull/2763/files#diff-eadede6f594353cfaf24138e4b22335b58e711eff9ca3121a84d67c5fda19bb5R84 to similarly recommend setting `analytics: false` in user-level configuration instead.